mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/documentdialog/DocumentDialogFragment.java

```diff
@@ -173,12 +173,17 @@ public void validateInput() throws RequiredFieldException {
                     getString(R.string.message_field_required));
 
         //Start Uploading Document
-        if (documentAction.equals(getResources().getString(R.string.update_document))) {
-            mDocumentDialogPresenter.updateDocument(entityType, entityId, document.getId(),
-                    documentName, documentDescription, fileChoosen);
-        } else if (documentAction.equals(getResources().getString(R.string.upload_document))) {
-            mDocumentDialogPresenter.createDocument(entityType, entityId,
-                    documentName, documentDescription, fileChoosen);
+        if (!(tvChooseFile.getText().length() == 0 ||
+                tvChooseFile.getText().toString().equals(""))) {
+            if (documentAction.equals(getResources().getString(R.string.update_document))) {
+                mDocumentDialogPresenter.updateDocument(entityType, entityId, document.getId(),
+                        documentName, documentDescription, fileChoosen);
+            } else if (documentAction.equals(getResources().getString(R.string.upload_document))) {
+                mDocumentDialogPresenter.createDocument(entityType, entityId,
+                        documentName, documentDescription, fileChoosen);
+            }
+        } else {
+            Toast.makeText(getContext(), "Add Document to upload!", Toast.LENGTH_SHORT).show();
```

Comment:
@crowded-geek Please add the toast text("Add Document to upload") to the string.xml file in order to support localization.

Reply:
Can I solve this ?